client/admin.go

```diff
@@ -12,3 +16,16 @@ func (c *APIClient) ReadConfig() (*models.APIConfig, error) {
 
 	return config, nil
 }
+
+func (c *APIClient) ReadAdminLogs(query url.Values) ([]models.LogFile, error) {
+	if _, _, _, err := controllers.ParseLoggingQuery(query); err != nil {
```

Comment:
This introduces a dependency between the `client` package on the `controller` package, which should at least be the other way around (or not exist at all). As a quick fix, `ParseLoggingQuery` should exist in this package.

Reply:
Should we move the function from the client package and then make calls in `controller` thus use `client.ParseLoggingQuery`?

Reply:
yep!